### PR TITLE
GUI image: optimize picking when detectPointerOnOpaqueOnly is true

### LINKED ISSUE #8652: GUI: Improve Image.contains performance when detectPointerOnOpaqueOnly is true
See forum post https://forum.babylonjs.com/t/image-class-has-poor-performance-when-transparent-parts-of-image-should-be-ignored-detectpointeronopaqueonly/13018

### PR DESCRIPTION
**Pull request by Popov72**
Fix #8652

### CODE REVIEW

File: gui/src/2D/controls/image.ts [hunk 1]
```diff
@@ -632,16 +641,23 @@ export class Image extends Control {
             return true;
         }
 
-        const canvas = this._workingCanvas;
-        const context = canvas.getContext("2d")!;
         const width = this._currentMeasure.width | 0;
         const height = this._currentMeasure.height | 0;
-        const imageData = context.getImageData(0, 0, width, height).data;
+        const key = width + "_" + height;
```
Comment (deltakosh) [inline gui/src/2D/controls/image.ts:646]: smart ;)

**Comment by deltakosh** [comment]
One test is failing

**Comment by Popov72** [comment]
It's "PBR shader code coverage #2" that fails in WebGL1: there's no reason, this PR is not related to PBR...

I have checked on my local computer by running this test in WebGL 1 mode, and it is ok.

Did something change lately on the Azure plateform relative to WebGL 1 rendering?

**Comment by deltakosh** [comment]
Ok let me restart the tests

**Comment by Popov72** [comment]
All good!

**Pull request merged**